ycms/cms/urls/protected.py

```diff
@@ -3,11 +3,12 @@
 """
 from django.urls import include, path
 
-from ..views import authentication, index, patients, ward
+from ..views import authentication, index, patients, ward, zentral
 from ..views.utility.autocomplete import autocomplete_icd10, autocomplete_patient
 
 urlpatterns = [
     path("", index.UserBasedRedirectView.as_view(), name="index"),
+    path("zentral/", zentral.ZentralView.as_view(), name="zentral"),
```

Comment:
```suggestion
    path("ward/list/", zentral.ZentralView.as_view(), name="ward_list"),
```
Another option would be `ward/manage/` as a route and `ward_management` as a name. In both cases, please move this line to the bottom of the file, newar the other `ward/` routes.

THe reason I would suggest renaming and moving this endpoint is that ZBM/zentral are not really functions, ZBM is just a user role we have, and generally speaking it makes more sense to name views and routes based on their functionality, not who will be able to use the functionality.